photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java

```diff
@@ -437,6 +451,71 @@ private Optional<EstimatedRobotPose> multiTagOnCoprocStrategy(
         }
     }
 
+    private Optional<EstimatedRobotPose> imuTagOnRioStrategy(
+            PhotonPipelineResult result, Optional<Matrix<N3, N3>> cameraMatrixOpt) {
```

Comment:
Is there a difference from all the logic here to [this PhotonUtils function](https://github.com/PhotonVision/photonvision/blob/master/photon-lib/src/main/java/org/photonvision/PhotonUtils.java#L96-L115)? If not, why not use it?

Reply:
I don't think there's any major differences, so I'm not entirely sure why I decided to write it on my own. The PhotonUtils function seems to do what I want, so I'll rewrite imuTagOnRioStrategy to use it. Thank you for pointing that out, I'm not sure I realized it was there.

Reply:
It seems like the difference is in `estimateCameraToTargetTranslation`, where you put the observed tag pitch plus the camera pitch, and that doesn't. I believe you way is the right way, if I'm not mistaken

Reply:
One thing that's werid to me about that utils function, is how it works when the camera yaw isn't the gyro yaw. The only place where the robotToCamera yaw is usrd is in the end, but the only thing that'll do is change our angle. The field position would still be wrong (from the [transform here](https://github.com/PhotonVision/photonvision/blob/ed6cf0f5dc55ba4a643d7ab49f5f5c5326de934b/photon-lib/src/main/java/org/photonvision/PhotonUtils.java#L129))
Maybe that transform needs to also take the robotToCamera yaw into account? I may be missing something.